libs/langchain/langchain/vectorstores/pinecone.py

```diff
@@ -303,6 +325,44 @@ def max_marginal_relevance_search(
             embedding, k, fetch_k, lambda_mult, filter, namespace
         )
 
+    @classmethod
+    def get_pinecone(
```

Comment:
:thinking: Should this be `get_or_create_pinecone_index`? I'm guessing it's not as pythonic if the name is too long? :smile: but I'm still ramping up. 

Wonder if making the name of the method even more explicit would help folks understand what it's doing and how it's intended to be used.

Reply:
Maybe `get_pinecone_index`, or `connect_pinecone_index` or something. I wouldn't write `create` here since it will be misleading.

Reply:
Good call :+1: